narwhals/_arrow/utils.py

```diff
@@ -246,3 +250,27 @@ def cast_for_truediv(arrow_array: Any, pa_object: Any) -> tuple[Any, Any]:
         )
 
     return arrow_array, pa_object
+
+
+def broadcast_series(series: list[ArrowSeries]) -> list[Any]:
+    lengths = [len(s) for s in series]
+    max_length = max(lengths)
+    fast_path = all(_len == max_length for _len in lengths)
+
+    if fast_path:
+        return [s._native_series for s in series]
+
+    pa = get_pyarrow()
+
+    reshaped = []
+    for s, length in zip(series, lengths):
+        s_native = s._native_series
+        if max_length > 1 and length == 1:
+            value = s_native[0]
+            if hasattr(value, "as_py"):  # pragma: no cover
+                value = value.as_py()
```

Comment:
does this need doing everywhere or just for `self._backend_version < (13,)`?

Reply:
Need to check, numpy was certainly failing to create an array with a value of arrow scalar